modules/metering-uninstall.adoc

```diff
@@ -3,25 +3,66 @@
 // * metering/metering-uninstall.adoc
 
 [id="metering-uninstall_{context}"]
-= Uninstalling metering from {product-title}
+= Uninstalling the Metering Operator from {product-title}
 
-You can remove metering from your cluster.
+You can remove the Metering Operator and the associated metering components from your cluster.
+
+[NOTE]
+====
+Metering does not manage or delete any S3 bucket data. If you used Amazon S3 for storage, any S3 buckets used to store metering data must be manually cleaned up.
+====
+
+== Removing the Metering Operator
+
+You can remove the Metering Operator by following the documentation on xref:../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
+
+== Uninstalling metering
+
+To uninstall metering, you must remove the MeteringConfig resource and delete the `openshift-metering` namespace.
```

Comment:
@lbarbeevargas is there a doc standard as to when to use resource vs. custom resource?

Reply:
Also, do you know if MeteringConfig -> `MeteringConfig` here?

Reply:
@timflannagan1 I'm checking on the question regarding resource vs custom resource. Based on the [Installing metering](https://docs.openshift.com/container-platform/4.5/metering/metering-installing-metering.html) docs, I'm predominantly seeing "MeteringConfig resource." 

Also, with the exception of [About configuring metering](https://docs.openshift.com/container-platform/4.5/metering/configuring_metering/metering-about-configuring.html), MeteringConfig does not have backticks.